.gitignore

```diff
@@ -7,3 +7,4 @@ npm-debug.log
 .deps-tmp
 /coverage
 *.pem
+.esm-cache
```

Comment:
Should this also be added to `.npmignore`?

Reply:
No need. Npm already ignores files listed in `.gitignore`.

Reply:
Hmm.  I don't think so? I just re-read through https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package and it seems that `.gitignore` is only used if you don't already have a `.npmignore`.  In our case, we do have an `.npmignore` so we will also need to update it.

Reply:
Wow, you are right. This `.ignore` stuff is quite awful. I'm sure there are tons of packages on the npm repo with fun secrets that have accentually been included.

Reply:
You are very correct...